pandas/plotting/_core.py

```diff
@@ -861,11 +863,20 @@ def _plot_colorbar(self, ax, **kwds):
 class ScatterPlot(PlanePlot):
     _kind = 'scatter'
 
-    def __init__(self, data, x, y, s=None, c=None, **kwargs):
+    def __init__(self, data, x, y, s=None, c=None, size_factor=1, **kwargs):
         if s is None:
-            # hide the matplotlib default for size, in case we want to change
-            # the handling of this argument later
-            s = 20
+            # Set default size if no argument is given.
+            s = 20 * size_factor
+        elif is_hashable(s) and s in data.columns:
+            # Handle the case where s is a label of a column of the df.
+            # The data is normalized to 200 * size_factor.
+            self.size_title = s
+            self.bubble_points = 200
+            self.size_factor = size_factor
+            size_data = data[s]
```

Comment:
What happens `s` is a label with duplicates? This would be a dataframe. We should error gracefully in that case.